db-run.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+
+# Loads various settings that are used elsewhere in the script
+# This should be called before any other functions
+docker_setup_env() {
+	declare -g DATABASE_ALREADY_EXISTS
+	# look specifically for PG_VERSION, as it is expected in the DB dir
+	if [ -s "$PGDATA/PG_VERSION" ]; then
```

Comment:
This test is defective. 
- Missing a `$`
- `$PG_VERSION` is not set.

Wouldn't the major version suffice anyway?

Reply:
No, the test is correct. `PG_VERSION` is a file inside the data directory which is created when the cluster created.